app/views/other/aeoi/haveYouRegisteredAEOI.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+@import uk.gov.hmrc.play.views.html._
+@import controllers.other.aeoi.routes._
+@import models.other.aeoi.HaveYouRegisteredAEOI
+
+
+@(appConfig: FrontendAppConfig, form: Form[_])(serviceInfoContent: Html)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("haveYouRegisteredAEOI.title"),
+    appConfig = appConfig,
+bodyClasses = None,
```

Comment:
tab code in

Reply:
have done it.. I will push when I merge master last time after all PR's are merged.. thank you